src/hosts/Elsa.Studio.Host.CustomElements/wwwroot/index.html

```diff
@@ -24,7 +24,11 @@
 </head>
 
 <body>
-<elsa-studio-workflow-definition-editor definition-id="c6deed7e9c4847e8af0647109781e1f3"></elsa-studio-workflow-definition-editor>
+<elsa-backend-provider remote-endpoint="https://localhost:5001/elsa/api" access-token="eyJhbGciOiJodHRwOi8vd3d3LnczLm9yZy8yMDAxLzA0L3htbGRzaWctbW9yZSNobWFjLXNoYTI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJodHRwOi8vZWxzYS5hcGkiLCJpc3MiOiJodHRwOi8vZWxzYS5hcGkiLCJleHAiOjE3MzQyMDgxMTUsImlhdCI6MTczNDEyMTcxNSwibmFtZSI6ImFkbWluIiwicGVybWlzc2lvbnMiOiIqIiwicm9sZSI6IkFkbWluaXN0cmF0b3IiLCJuYmYiOjE3MzQxMjE3MTV9.YkIlgdxOHDRZhxlmrSvs-AlL5XKlTuI8yGQZtltdNZ4"></elsa-backend-provider>
```

Comment:
I don't have a solution in mind yet, but I'm not sure it's a good idea to include an access token here in the source code.